extism.go

```diff
@@ -23,6 +23,9 @@ import (
 //go:embed extism-runtime.wasm
 var extismRuntimeWasm []byte
 
+//go:embed extism-runtime.wasm.version
+var ExtismRuntimeWasmVersion string
```

Comment:
is it possible to make this a const instead? and if not, can we make the name `extismRuntimeWasmVersion` so it is not mutable by any thing that imports it?

If you need a public getter, also feel free to add a function that reads and returns the value.